libs/deposits/src/lib/deposit-form.tsx

```diff
@@ -176,25 +177,38 @@ export const DepositForm = ({
             {errors.asset.message}
           </InputError>
         )}
-        {isFaucetable && selectedAsset && (
-          <UseButton onClick={requestFaucet}>
-            {t(`Get ${selectedAsset.symbol}`)}
-          </UseButton>
-        )}
         {!errors.asset?.message && selectedAsset && (
-          <button
+          <UseButton
             data-testid="view-asset-details"
-            className="text-sm underline"
             onClick={() => {
               setAssetDetailsDialogOpen(true);
               setAssetDetailsDialogSymbol(selectedAsset);
             }}
           >
             {t('View asset details')}
-          </button>
+          </UseButton>
+        )}
+        {isFaucetable && selectedAsset && (
+          <span className="absolute bottom-0 right-0">
```

Comment:
Chore (blocking): I can't see an element that these `absolutes` are `relative` to. Different browsers can interpret differently, you can get wildly different results based on whether the browser assumes a parent element or the html element. If I'm wrong then apologies and ignore, but if there is no `relative` positioning on the parent element, please set it.

Reply:
They all have `<FormGroup>` as a parent, which are set to `relative` by default:

![image](https://user-images.githubusercontent.com/1325606/191790116-f8a8a1f5-4a48-4c2a-b796-3e1183aba330.png)